src/ng-generate/prompter/prompts-questions/shared/prompt-complex-questions.ts

```diff
@@ -510,3 +510,86 @@ export async function getDatePickerType(templateHelper: TemplateHelper, allAnswe
 async function datePickerTypePrompt(property: Property): Promise<any> {
     return inquirer.prompt([requestChooseDatePickerType(property)]);
 }
+
+function getFilterProperties(templateHelper: TemplateHelper, allAnswers: any,answers: any, aspect: Aspect, enabledCommandBarFunctions?:any[]): string[]{
+    const hasEnumFilter =  enabledCommandBarFunctions ? enabledCommandBarFunctions.includes('addEnumQuickFilters') : false;
+    const hasDateFilter =  enabledCommandBarFunctions ? enabledCommandBarFunctions.includes('addDateQuickFilters') : false;
+    allAnswers.selectedModelElementUrn = answers.selectedModelElementUrn || templateHelper.resolveType(aspect).aspectModelUrn;
+    const props = templateHelper.getProperties(
+        {
+            selectedModelElement: loader.findByUrn(allAnswers.selectedModelElementUrn),
+            excludedProperties: [],
+        }
+      );
+     
+      const filterProps:string[] = [];
+      props.forEach((prop:Property)=>{
+         if(hasEnumFilter && templateHelper.isEnumProperty(prop)){
+            filterProps.push(prop.name);
+         } else if(hasDateFilter && (templateHelper.isDateProperty(prop) || templateHelper.isDateTimeProperty(prop) || templateHelper.isDateTimestampProperty(prop))){
+            filterProps.push(prop.name);
+         }
+      });
+
+      return filterProps;
+}
+
+async function commandBarFilterOrderPrompt(templateHelper: TemplateHelper, allAnswers: any,answers: any, aspect: Aspect, options: Schema,enabledCommandBarFunctions?:any[]): Promise<any> { 
+    const choices = getFilterProperties(templateHelper, allAnswers, answers, aspect, enabledCommandBarFunctions);
+    const orderedChoices = await orderItems(choices);
+    
+    options.commandBarFilterOrder = orderedChoices;
+    allAnswers['commandBarFilterOrder'] = orderedChoices;
+  }
+
+export async function getCommandBarFilterOrder(templateHelper: TemplateHelper, allAnswers: any, answers: any, aspect: Aspect, options:Schema,enabledCommandBarFunctions:any[]): Promise<object> {
+    return await commandBarFilterOrderPrompt(templateHelper,allAnswers,answers,aspect, options,enabledCommandBarFunctions);
+}
+
+async function orderItems(items:any) {
+    const orderedItems = [...items];
+    let exit = false;
+  
+    while (!exit) {
+      console.log('Order the command bar filters:');
+      orderedItems.forEach((item, index) => {
+        console.log(`${index + 1}. ${item}`);
```

Comment:
Also this.

Reply:
i need this, it is used when the filter properties are shown